docs/usage/configuration.rst

```diff
@@ -113,3 +113,17 @@ Any other field definition will not be checked.
 .. literalinclude:: /examples/configuration/test_example_8.py
     :caption: Enable Check Factory Fields
     :language: python
+
+Use Default Values
+------------------
+
+If ``__use_default__`` is set to ``True``, then the default value will be used instead of creating a random value
+for a given field, provided there's a default value for that field.
+
+By default, ``__use_default__`` is set to ``False.`` If you need more fine grained control, you can override the
+:meth:`BaseFactory <polyfactory.factories.base.BaseFactory.should_use_default_value>` classmethod.
```

Comment:
@JacobCoffee this is getting rendered as `BaseFactory` instead of `should_use_default_value` or at least `BaseFactory.should_use_default_value`. Do you know what I'm doing wrong?

Reply:
@JacobCoffee sorry for the double ping, but do you know how I can fix this? If not, I think I'll just write out the function name directly instead of linking to it.

Reply:
```rst
:meth:`~polyfactory.factories.base.BaseFactory.should_use_default_value` classmethod.
```

Gives me 
<img width="792" alt="image" src="https://github.com/litestar-org/polyfactory/assets/45509143/74653354-0701-4a15-ab98-1bcccd15f8b2">

Reply:
Oh that's great! Thanks @Alc-Alc :)